### ISSUE TITLE
Edge model: remove architecture-specific items

### ISSUE THREAD

**Issue by Kevsy** [camaraproject/EdgeCloud#81]
The [Edge Model](https://github.com/camaraproject/EdgeCloud/blob/main/documentation/SupportingDocuments/edgemodel.md) includes entities from the GSMA Operator Platform, namely `USER_CLIENT`, `OPERATOR_PLATFORM`, and (I think) `EDGE_CLOUD`.

Suggest to remove `USER_CLIENT` because it is an architecture detail for operators deploying the GSMA operator platform but not part of the API model. It can still be included in the model but as an abstraction that covers both direct communication or communication via a user client.

Suggest to rename 'OPERATOR_PLATFORM' to `NETWORK_OPERATOR_API_GATEWAY`.

Do we need EDGE_CLOUD in the model, or is it GSMA Operator Platform specific?

**Comment by ThomasEdgeXR** [comment]
I agree with the issue. The Edge Model follows the entities listed in https://github.com/camaraproject/EdgeCloud/blob/main/documentation/SupportingDocuments/edge_terminology.md. I agree we should converge here towards abstract definitions independent of a specific architecture or heritage. I'd therefore suggest to first fix the edge terminology to list all entities in an architecture independent way.

In matter i would not agree with renaming OP to API Gateway as the OP covers substantially more roles than an API Gateway.
